jobs/README.md

```diff
@@ -4,6 +4,12 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Front End Developers @ [Ecoworld SA – Greekferries Club](https://greekferries-club.gr)
+Ecoworld SA is looking for 2 Front End Developers (Junior & Senior) for full-time, in-house employment
+to work on various innovative online-travel related projects 
```

Comment:
IIRC for a list to start, you need a blank line before the list items that start with `*`

Reply:
I didn't want to start a list. The new line was redundant though.

Reply:
please ignore. I was probably confusing asciidoc, with markdown.